impeller/typographer/glyph_atlas.h

```diff
@@ -19,6 +19,16 @@ namespace impeller {
 
 class FontGlyphAtlas;
 
+struct FrameBounds {
+  /// The bounds of the glyph within the glyph atlas.
+  Rect atlas_bounds;
+  /// The local glyph bounds.
+  Rect glyph_bounds;
+  /// Whether [atlas_bounds] are still a placeholder and have
+  /// not yet been computed.
+  bool placeholder;
```

Comment:
Zero out the scalar.

Reply:
Name this `is_placeholder` instead? While reading this later, I thought placeholder returned a placeholder itself.

Reply:
Done